### PR TITLE
Rename SelectApplicationType to PreviousBTIRuling

### PR DESCRIPTION
**Pull request by DavidGregory084**

### CODE REVIEW

File: test/unit/controllers/SelectApplicationTypeControllerSpec.scala [hunk 1]
```diff
@@ -19,24 +19,24 @@ package controllers
...
 import play.api.libs.json.JsString
 import play.api.mvc.{ Call, Request }
-import views.html.selectApplicationType
+import views.html.previousBTIRuling
 import pages.ProvideGoodsNamePage
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class SelectApplicationTypeControllerSpec extends ControllerSpecBase with YesNoCachingControllerBehaviours {
+class PreviousBTIRulingControllerSpec extends ControllerSpecBase with YesNoCachingControllerBehaviours {
```
Comment (OC-Pomaroff) [inline test/unit/controllers/SelectApplicationTypeControllerSpec.scala:33]: This file should be renamed too

File: test/unit/forms/SelectApplicationTypeFormProviderSpec.scala [hunk 2]
```diff
@@ -19,14 +19,14 @@ package forms
 import forms.behaviours.BooleanFieldBehaviours
 import play.api.data.FormError
 
-class SelectApplicationTypeFormProviderSpec extends BooleanFieldBehaviours {
+class PreviousBTIRulingFormProviderSpec extends BooleanFieldBehaviours {
```
Comment (OC-Pomaroff) [inline test/unit/forms/SelectApplicationTypeFormProviderSpec.scala:22]: This file should be renamed

File: test/unit/views/SelectApplicationTypeViewSpec.scala [hunk 3]
```diff
@@ -18,31 +18,31 @@ package views
...
-import forms.SelectApplicationTypeFormProvider
+import forms.PreviousBTIRulingFormProvider
 import models.NormalMode
 import views.behaviours.ViewBehaviours
-import views.html.selectApplicationType
+import views.html.previousBTIRuling
 import views.behaviours.YesNoViewBehaviours
 
-class SelectApplicationTypeViewSpec extends YesNoViewBehaviours {
+class PreviousBTIRulingViewSpec extends YesNoViewBehaviours {
```
Comment (OC-Pomaroff) [inline test/unit/views/SelectApplicationTypeViewSpec.scala:27]: Also this file

**Review by OC-Pomaroff — APPROVED** [review]

**Comment by DavidGregory084** [comment]
Hahah remind me never to rely on the file name search in VS Code again 😂 I used fd to double check some of the other PRs

**Pull request merged**